### PR TITLE
This demo had erroneous modal message that was not dynamic

### PR DESCRIPTION
**Pull request by murattishkul**
added logic for dynamic modal text

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

-----
[View rendered components/modal/demo/async.md](https://github.com/murattishkul/ant-design/blob/patch-1/components/modal/demo/async.md)

### CODE REVIEW

File: components/modal/demo/async.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ class App extends React.Component {
           confirmLoading={confirmLoading}
           onCancel={this.handleCancel}
         >
-          <p>{ModalText}</p>
+          <p>{this.state.ModalText}</p>
```
Comment (xrkffgg) [inline components/modal/demo/async.md:66]: Look 53 line

Comment (murattishkul) [reply components/modal/demo/async.md:66]: @xrkffgg  haha sorry

Comment (murattishkul) [reply components/modal/demo/async.md:66]: > @xrkffgg haha sorry
what's wrong then? (https://ant.design/components/modal/)

Comment (xrkffgg) [reply components/modal/demo/async.md:66]: What‘s the problem?

Comment (murattishkul) [reply components/modal/demo/async.md:66]: > What‘s the problem?

No problem. I thought "Open Modal with async logic" example's modal text is not 'Content of the modal' first time it is open. I just refreshed the page and it is fine. Sorry to bother. thanks for collab

Comment (murattishkul) [reply components/modal/demo/async.md:66]: you may close this pr

**Pull request closed without merging**